Header header
string turtle_id
turtlesim/Pose pose
bool position_sent
bool arrived
